src/main/java/ch/cern/eam/wshub/core/services/workorders/entities/WorkOrderActivityCheckListDefault.java

```diff
@@ -0,0 +1,108 @@
+package ch.cern.eam.wshub.core.services.workorders.entities;
+
+import ch.cern.eam.wshub.core.annotations.InforField;
+import net.datastream.schemas.mp_fields.NOUN_Type;
+import net.datastream.schemas.mp_fields.VERB_Type;
+
+import java.io.Serializable;
+import java.math.BigInteger;
+
+public class WorkOrderActivityCheckListDefault implements Serializable {
```

Comment:
Is this class necessary? Why not use the class from the proxy client/the Infor class?

Reply:
Correct, the usage of this class can be avoided. Not using this class will lead to the code in getSignatureWS to be less concise. This is also supported by the fact that certain fields need to be set to default values in the Infor class.

Reply:
Removed